bottlecap/src/lifecycle/flush_control.rs

```diff
@@ -57,12 +52,15 @@ impl FlushControl {
                 i.set_missed_tick_behavior(Skip);
                 i
             }
+            // TODO: Why is this 15 minutes?
```

Comment:
Need help here

Reply:
Looking at the history/blame can be helpful here: https://github.com/DataDog/datadog-lambda-extension/pull/599

This timeout is used for the race flush block. But when users want to flush at end, we should never hit the race flush. Initially it was set to a max unsigned int but that caused tokio to integer overflow when calculating the timeout